toolkit/scripts/toolchain.mk

```diff
@@ -49,6 +49,8 @@ $(call create_folder,$(toolchain_downloads_logs_dir))
 $(call create_folder,$(toolchain_from_repos))
 $(call create_folder,$(populated_toolchain_chroot))
 
+##help:target:raw-toolchain=Build the initial toolchain bootstrap stage.
+##help:target:toolchain=Ensure all toolchain RPMs are present.
```

Comment:
This is one comment but applies to other occurrences in the PR:

What do you think about adding these comments directly above the target they refer to? This way it's going to be harder to "misplace" them if we move some targets around. For the sake of making it easier to read, I'm all for adding empty lines between different targets as well.